flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/utils/StreamingWithStateTestBase.scala

```diff
@@ -239,6 +239,7 @@ object StreamingWithStateTestBase {
     override def toString: String = backend.toString
   }
 
+  // TODO test FORSTDB_BACKEND after 2.0-preview
```

Comment:
We can create a jira to track this and avoid using the TODO annotation because there left too many staled TODOs

Reply:
I created FLINK-36157